src/staticanalysis/bot/tests/test_remote.py

```diff
@@ -86,13 +102,20 @@ def test_no_deps(mock_try_config, mock_revision):
     from static_analysis_bot.workflows.remote import RemoteWorkflow
 
     tasks = {
+        'decision': {
+            'image': 'taskcluster/decision:XXX',
+            'env': {
+                'GECKO_HEAD_REPOSITORY': 'https://hg.mozilla.org/try',
+                'GECKO_HEAD_REV': 'deadbeef1234',
+            }
+        },
         'remoteTryTask': {},
         'extra-task': {},
     }
     workflow = RemoteWorkflow(MockQueue(tasks))
     with pytest.raises(AssertionError) as e:
         workflow.run(mock_revision)
-        assert e.message == 'No task dependencies to analyze'
+    assert str(e.value) == 'No task dependencies to analyze'
```

Comment:
Nit: better to keep it with the previous indentation

Reply:
It does not work in the `with` block !
I noticed that while playing with the assertion messages and it makes sense: the exception is caught by the `with` statement so that the code below in the block's stack will not be executed.

Plus, here is a sample on the [official doc](https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions)